salt/modules/chocolatey.py

```diff
@@ -419,6 +420,9 @@ def install(name,
         package_args (str):
             Arguments you want to pass to the package. Default is None.
 
+        execution_timeout (str):
+        Chocolatey execution timeout value you want to pass to the installation process. Default is None.
+
```

Comment:
Can you add a `.. versionadded:: Oxygen` tag here?

Reply:
Done.